.vitepress/theme/components/AuthButtons/AuthButtons.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+defineProps<{ place: 'navScreen' | 'navBar' }>();
+</script>
+
+<template>
+  <div :class="place === 'navScreen' ? 'navScreenContainer' : 'navBarContainer'">
+    <a href="https://stackblitz.com/sign_in" class="link light">Sign in</a>
+    <a href="https://stackblitz.com/register" class="link accent">Get started</a>
+  </div>
+</template>
+
+<style scoped lang="scss">
+@import './linkStyles.scss';
+
+/**
+ * When we have enough space, we show auth buttons in nav bar.
+ * When we don't, we show them inside the hamburger menu.
+ */
+.navBarContainer {
+  margin-inline-start: 40px;
+  display: flex;
+  gap: 8px;
+  // this is when we hide top nav menu
+  @media screen and (max-width: 768px) {
+    margin-inline-start: 0px;
+  }
+  @media screen and (max-width: 400px) {
```

Comment:
```suggestion
  @media screen and (max-width: 430px) {
```

`430px` to prevent this

https://github.com/stackblitz/docs/assets/6536985/530d7a21-f363-4590-a7ad-93971e8f7e3e